dev/TeachingTip/TeachingTip.cpp

```diff
@@ -740,98 +808,105 @@ void TeachingTip::UpdateDynamicHeroContentPlacementToBottom()
 
 void TeachingTip::OnIsOpenChanged()
 {
-    m_acceleratorKeyActivatedRevoker = {};
-
-    if (IsOpen())
+    auto const strongThis = get_strong();
+    SharedHelpers::QueueCallbackForCompositionRendering([strongThis]() 
     {
```

Comment:
I *believe* you can do:
```suggestion
    SharedHelpers::QueueCallbackForCompositionRendering([strongThis = get_strong()]() 
```

And remove the explicit local.